docs/pages/authentication/access-token/nextjs.mdx

```diff
@@ -0,0 +1,207 @@
+---
+meta:
+  title: "Set up access token permissions with Next.js"
+  parentTitle: "Authentication"
+  description: "Learn how to setup access token permissions with Next.js."
+---
+
+Follow the following steps to start configure your authentication endpoint and
+start building your own security logic in Next.js’ `/app` directory.
+
+## Quickstart
+
+<Steps>
+  <Step>
+    <StepTitle>Install the `liveblocks/node` package</StepTitle>
+    <StepContent>
+
+      ```bash
+      npm install @liveblocks/node
+      ```
+
+    </StepContent>
+
+  </Step>
+  <Step>
+    <StepTitle>Set up authentication endpoint</StepTitle>
+    <StepContent>
+      Users can only interact with rooms they have access to. You can
+      configure permission access in an `api/liveblocks-auth` endpoint by
+      creating the `app/api/liveblocks-auth/route.ts` file with the
+      following code. This is where you will implement your security and
+      define the rooms that the user has access to.
+
+      The client will request a token either for a specific room, in which case the
+      request body property `room` will be defined, or for resources outside a
+      specific room (like notifications), in which case, `room` will be undefined.
+
+      See
+      [permissions with access token](/docs/rooms/permissions/access-token)
+      to understand how to grant access to multiple rooms with an access token.
+
+      ```ts file="app/api/liveblocks-auth/route.ts"
+      import { Liveblocks } from "@liveblocks/node";
+
+      const liveblocks = new Liveblocks({
+        secret: "{{SECRET_KEY}}",
+      });
+
+      export async function POST(request: Request) {
+        // Get the current user from your database
+        const user = __getUserFromDB__(request);
+
+        // Start an auth session inside your endpoint
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.metadata } // Optional
+        );
+
+        // Implement your own security, and give the user access to the room/organization.
+        // Note: Even if room is defined, we recommend to always use wildcards.
+        const { room } = request.body;
```

Comment:
Should we use wildcards as the main example (even if room is set)?

And add another section to show how to generate token per room/